doc/release/1.3.0.md

```diff
@@ -3,5 +3,31 @@
 
 ## Notes about new/updated Features
 
+### Forger reward address
+For forger nodes: it is now possible to specify a custom address where the block reward will be directed.<br> 
+The address will be put inside every new block forged by the node.<br> 
+(Previously the first address on the local wallet was taken automatically, without the possibility to specify a different one).<br> 
+Any EON address can be specified (included a smart contract address). The private key does not need to be imported in the local wallet.
+
+How to specify it: 
+- inside the config file, add a property like this inside the *forger* section:
+
+    forgerRewardAddress = "xxxxxxxxx"
+
+- if using docker, you can just configure the env property SCNODE_FORGER_REWARD_ADDRESS: the config file will be automatically generated accordingly.
+
+### Pause forging
+Please be aware that starting from this version EON block production may temporary pause if the mainchain block production slows down: EON forgers will automatically stop to produce new blocks if in the latest 99 EON Blocks any reference to mainchain blocks is present.
```

Comment:
Shouldn't be "if in the latest 99 EON Blocks no reference to mainchain blocks is present"?